language_service/src/completion/tests.rs

```diff
@@ -40,31 +36,135 @@ fn in_block_contains_std_functions() {
         &expect![[r#"
             [
                 Some(
-                    (
-                        "Fake",
-                        Function,
-                        Some(
+                    CompletionItem {
```

Comment:
I'd like to have these expects be a bit more succinct, but I've found it difficult to have something like a test-specific Display/Debug impl for the data types involved. @samarsha @swernli, any suggestions or Rust-tips to make this readout simpler?